starlette_web/common/app.py

```diff
@@ -69,7 +69,7 @@ def get_debug(self) -> bool:
         return settings.APP_DEBUG
 
     def get_middleware(self) -> List[Middleware]:
```

Comment:
можем сделать во множ.числе? у тебя метод возвращает список `Middleware` же:)
```suggestion
    def get_middlewares(self) -> List[Middleware]:
```

ну и небольшой ренейм самих сеттингов: `MIDDLEWARE` -> `MIDDLEWARES`

Reply:
готово